.gitignore

```diff
@@ -26,3 +26,5 @@ nb-configuration.xml
 # OSX
 .DS_Store
 release-pom.xml
+
+/.mvn/wrapper/maven-wrapper.jar
```

Comment:
you'll want to add probably `.mvn/wrapper/*.ja` to capture any named jar or java files in here, and then delete `MavenWrapperDownloader.java` from this branch as well

Reply:
thanks!

Reply:
Finally I had to commit the file because of an error in Travis build:

```
--2021-09-08 21:03:25--  https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.jar
Resolving repo.maven.apache.org (repo.maven.apache.org)... 199.232.192.215, 199.232.196.215
Connecting to repo.maven.apache.org (repo.maven.apache.org)|199.232.192.215|:443... connected.
ERROR: cannot verify repo.maven.apache.org's certificate, issued by 'CN=GlobalSign Atlas R3 DV TLS CA H2 2021,O=GlobalSign nv-sa,C=BE':
  Unable to locally verify the issuer's authority.
To connect to repo.maven.apache.org insecurely, use `--no-check-certificate'.
```